pkg/parse/tests/foreign_import_swagger.sysl.golden.textpb

```diff
@@ -131,6 +131,16 @@ apps: <
           query_param: <
             name: "depth"
             type: <
+              type_ref: <
```

Comment:
These changes ( golden.textpb changes without corresponding .sysl changes) are regressions and not compatible with legacy python.
Need to be looked at more closely.

Reply:
compatibility is not a priority, anz-bank/sysl-legacy is where that belongs.